app/src/main/java/com/mk/tasky/agenda/detail/components/DetailDescription.kt

```diff
@@ -0,0 +1,57 @@
+package com.mk.tasky.agenda.detail.components
+
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.material.Icon
+import androidx.compose.material.Text
+import androidx.compose.material.icons.Icons
+import androidx.compose.material.icons.filled.NavigateNext
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import com.mk.tasky.R
+import com.mk.tasky.ui.theme.Black
+
+@Composable
+fun DetailDescription(
+    description: String,
+    isEditable: Boolean,
+    onClick: () -> Unit,
+    modifier: Modifier = Modifier
+) {
+    Row(
+        modifier = modifier.fillMaxWidth().clickable {
+            if (isEditable) {
```

Comment:
Here, it would make sense to only add the clickable modifier if it's editable, since else you'll get the ripple click effect even though it's not clickable.
Something like this:
```
modifier.then(
    if(isEditable) Modifier.clickable {
        onClick()
    } else Modifier
)
```

Reply:
Game changer. Thanks!